app/views/shared/_category_card.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="category">
+  <div class="category_card">
+    <div class="icon_card">
+      <%= image_tag(category.icon, class: "icon") if category.icon.attached? %>
+    </div>
+    <div class= "category_info">
+      <%= category.name %>
+      <div class="category_transaction">
+        $<%= category.sum_total %>
+      </div>
+    </div>
```

Comment:
- At the top of the page the user could see the total amount for the category (sum of all of the amounts of the transactions in that category). Now the sum of your transactions are not calculated. Kindly work on it.

Your design:
![Screenshot from 2022-08-13 23-47-11](https://user-images.githubusercontent.com/65860678/184511935-1b9b1ef8-90cb-4a59-9391-f224e58d30b1.png)

Given design:
![Screenshot from 2022-08-13 23-48-41](https://user-images.githubusercontent.com/65860678/184511941-1aa21fe6-c42f-494a-9bf9-fda6b645380d.png)